tools/regz/src/gen.zig

```diff
@@ -770,6 +772,12 @@ fn write_register(
     if (register.description) |description|
         try write_doc_comment(arena, description, writer);
 
+    // Add offset comment
```

Comment:
I could put this on the same line as the name, e.g. `/// I2C_SCL_LOW_PERIOD_REG (offset: 0x0)` if you think that looks nicer.